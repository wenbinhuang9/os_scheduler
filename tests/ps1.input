1 0 6 50
2 0 8 50
3 0 7 60
4 0 3 10