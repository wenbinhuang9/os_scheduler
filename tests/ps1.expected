1 6 0 6
2 8 0 8
3 15 8 15
4 11 8 11
4 10